### PR TITLE
Operator version 3.1.2

### PR DESCRIPTION
**Pull request by rjeberhard**
This release resolves an issue where the operator failed to start servers where the pods were configured to have an annotation containing a forward slash.

### CODE REVIEW

**Review by alai8 — APPROVED** [review]

File: docs-source/content/release-notes.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ draft: false
 
 | Date | Version | Introduces backward incompatibilities? | Change |
 | --- | --- | --- | --- |
+| January 22, 2021 | v3.1.2 | no | Resolved an issue where the operator failed to start servers where the pods were configured to have an annotation containing a forward slash. |
```
Comment (rosemarymarano) [inline docs-source/content/release-notes.md:11]: Purely, a slight re-wording suggestion but what you have is fine also: Resolved an issue where the operator failed to start servers in which the pods were configured to have an annotation containing a forward slash.

**Review by rosemarymarano — APPROVED** [review]
LGTM

File: docs-source/content/release-notes.md (hunk 1 above)
Comment (rjeberhard) [reply docs-source/content/release-notes.md]: +1

File: docs-source/content/quickstart/get-images.md [hunk 2]
```diff
@@ -38,8 +29,8 @@ and accept the license agreement for the [WebLogic Server image](https://hub.doc
      $ docker pull container-registry.oracle.com/middleware/weblogic:12.2.1.4
      ```
 
-    {{% notice note %}} The WebLogic Server Docker image, `weblogic:12.2.1.3`, has all the necessary patches applied. The WebLogic Server Docker image, `weblogic:12.2.1.4`, does not require any additional patches.
+    {{% notice note %}} The WebLogic Server image, `weblogic:12.2.1.3`, has all the necessary patches applied. The WebLogic Server Docker image, `weblogic:12.2.1.4`, does not require any additional patches.
```
Comment (rosemarymarano) [inline docs-source/content/quickstart/get-images.md:32]: The WebLogic Server Docker image, `weblogic:12.2.1.4`, -> The WebLogic Server image,, `weblogic:12.2.1.4`,

File: docs-source/content/samples/simple/domains/domain-home-on-pv/_index.md [hunk 3]
```diff
@@ -112,9 +112,9 @@ The following parameters can be provided in the inputs file.
...
-| `imagePullPolicy` | WebLogic Docker image pull policy. Legal values are `IfNotPresent`, `Always`, or `Never` | `IfNotPresent` |
-| `imagePullSecretName` | Name of the Kubernetes Secret to access the Docker Store to pull the WebLogic Server Docker image. The presence of the secret will be validated when this parameter is specified |  |
+| `image` | WebLogic Server image. The operator requires either Oracle WebLogic Server 12.2.1.3.0 with patch 29135930 applied, or Oracle WebLogic Server 12.2.1.4.0, or Oracle WebLogic Server 14.1.1.0.0. The existing WebLogic Docker image, `container-registry.oracle.com/middleware/weblogic:12.2.1.3`, has all the necessary patches applied. For details on how to obtain or create the image, see [WebLogic Docker images]({{< relref "/userguide/managing-domains/domain-in-image/base-images/_index.md#creating-or-obtaining-weblogic-docker-images" >}}). | `container-registry.oracle.com/middleware/weblogic:12.2.1.3` |
```
Comment (rosemarymarano) [inline docs-source/content/samples/simple/domains/domain-home-on-pv/_index.md:115]: I think that this cross-reference, see [WebLogic Docker images]({{< relref "/userguide/managing-domains/domain-in-image/base-images/_index.md#creating-or-obtaining-weblogic-docker-images" >}}), was changed to Creating or obtaining WebLogic Server images.

File: docs-source/content/samples/simple/domains/domain-home-in-image/_index.md [hunk 4]
```diff
@@ -14,7 +14,7 @@ Before you begin, read this document, [Domain resource]({{< relref "/userguide/m
...
 
 * The WebLogic Deploy Tooling (WDT) sample requires that `JAVA_HOME` is set to a Java JDK version 1.8 or later.
-* The operator requires either Oracle WebLogic Server 12.2.1.3.0 with patch 29135930 applied, or Oracle WebLogic Server 12.2.1.4.0, or Oracle WebLogic Server 14.1.1.0.0. The existing WebLogic Docker image, `container-registry.oracle.com/middleware/weblogic:12.2.1.3`, has all the necessary patches applied. For details on how to obtain or create the image, see [WebLogic Docker images]({{< relref "/userguide/managing-domains/domain-in-image/base-images/_index.md#creating-or-obtaining-weblogic-docker-images" >}}).
+* The operator requires either Oracle WebLogic Server 12.2.1.3.0 with patch 29135930 applied, or Oracle WebLogic Server 12.2.1.4.0, or Oracle WebLogic Server 14.1.1.0.0. The existing WebLogic Server image, `container-registry.oracle.com/middleware/weblogic:12.2.1.3`, has all the necessary patches applied. For details on how to obtain or create the image, see [WebLogic Docker images]({{< relref "/userguide/managing-domains/domain-in-image/base-images/_index.md#creating-or-obtaining-weblogic-docker-images" >}}).
```
Comment (rosemarymarano) [inline docs-source/content/samples/simple/domains/domain-home-in-image/_index.md:17]: I think that this cross-reference, see [WebLogic Docker images]({{< relref "/userguide/managing-domains/domain-in-image/base-images/_index.md#creating-or-obtaining-weblogic-docker-images" >}}), was changed to Creating or obtaining WebLogic Server images.

File: docs-source/content/samples/simple/domains/domain-home-in-image/_index.md [hunk 5]
```diff
@@ -136,16 +136,16 @@ The following parameters can be provided in the inputs file.
...
-| `domainHomeImageBuildPath` | Location of the WebLogic "domain home in image" Docker image in the `https://github.com/oracle/docker-images.git` project. If not specified, use `./docker-images/OracleWebLogic/samples/12213-domain-home-in-image`. Another possible value is `./docker-images/OracleWebLogic/samples/12213-domain-home-in-image-wdt` which uses WDT, instead of WLST, to generate the domain configuration. | `./docker-images/OracleWebLogic/samples/12213-domain-home-in-image` |
+| `domainHomeImageBase` | Base WebLogic binary image used to build the WebLogic domain image. The operator requires either Oracle WebLogic Server 12.2.1.3.0 with patch 29135930 applied, or Oracle WebLogic Server 12.2.1.4.0, or Oracle WebLogic Server 14.1.1.0.0. The existing WebLogic Server image, `container-registry.oracle.com/middleware/weblogic:12.2.1.3`, has all the necessary patches applied. For details on how to obtain or create the image, see [WebLogic Docker images]({{< relref "/userguide/managing-domains/domain-in-image/base-images/_index.md#creating-or-obtaining-weblogic-docker-images" >}}). | `container-registry.oracle.com/middleware/weblogic:12.2.1.3` |
```
Comment (rosemarymarano) [inline docs-source/content/samples/simple/domains/domain-home-in-image/_index.md:139]: I think that this cross-reference, see [WebLogic Docker images]({{< relref "/userguide/managing-domains/domain-in-image/base-images/_index.md#creating-or-obtaining-weblogic-docker-images" >}}), was changed to Creating or obtaining WebLogic Server images.

File: docs-source/content/samples/simple/domains/domain-home-on-pv/_index.md [hunk 6]
```diff
@@ -15,7 +15,7 @@ Before you begin, read this document, [Domain resource]({{< relref "/userguide/m
 The following prerequisites must be met prior to running the create domain script:
 
 * Make sure the WebLogic Server Kubernetes Operator is running.
-* The operator requires either Oracle WebLogic Server 12.2.1.3.0 with patch 29135930 applied, or Oracle WebLogic Server 12.2.1.4.0, or Oracle WebLogic Server 14.1.1.0.0. The existing WebLogic Docker image, `container-registry.oracle.com/middleware/weblogic:12.2.1.3`, has all the necessary patches applied. For details on how to obtain or create the image, see [WebLogic Docker images]({{< relref "/userguide/managing-domains/domain-in-image/base-images/_index.md#creating-or-obtaining-weblogic-docker-images" >}}).
+* The operator requires either Oracle WebLogic Server 12.2.1.3.0 with patch 29135930 applied, or Oracle WebLogic Server 12.2.1.4.0, or Oracle WebLogic Server 14.1.1.0.0. The existing WebLogic Server image, `container-registry.oracle.com/middleware/weblogic:12.2.1.3`, has all the necessary patches applied. For details on how to obtain or create the image, see [WebLogic Docker images]({{< relref "/userguide/managing-domains/domain-in-image/base-images/_index.md#creating-or-obtaining-weblogic-docker-images" >}}).
```
Comment (rosemarymarano) [inline docs-source/content/samples/simple/domains/domain-home-on-pv/_index.md:18]: I think that this cross-reference, see [WebLogic Docker images]({{< relref "/userguide/managing-domains/domain-in-image/base-images/_index.md#creating-or-obtaining-weblogic-docker-images" >}}), was changed to Creating or obtaining WebLogic Server images.

**Review by rosemarymarano — COMMENTED** [review]
OMG, that's a lot of text to review/change. I caught only a few misses, but I will re-review this version when it goes out.

**Pull request merged**